edb/server/compiler/sql.py

```diff
@@ -293,13 +293,95 @@ def _compile_sql(
                 from edb.pgsql import resolver as pg_resolver
                 pg_resolver.dispatch._raise_unsupported(stmt)
 
-            unit.get_var = stmt.name
-            unit.frontend_only = (
-                stmt.name in FE_SETTINGS_MUTABLE
-                or stmt.name.startswith('global ')
+            pg_settings_for_show = pgast.RelRangeVar(
```

Comment:
Can this all go in a sub function? It is getting pretty large.

Reply:
Yeah, that sounds like a good idea, let me fix it